contract/src/pegasus.js

```diff
@@ -96,6 +97,7 @@ function makeICS20Converter(localBrand, prefixedDenom) {
       brand === localBrand,
       details`Brand must our local issuer's, not ${q(brand)}`,
     );
+    // @ts-ignore
```

Comment:
Please always append a reason comment to the ts-ignore line so I understand why.